corehq/apps/data_dictionary/static/data_dictionary/js/data_dictionary.js

```diff
@@ -78,9 +83,30 @@ hqDefine("data_dictionary/js/data_dictionary", [
             self.removeFHIRResourcePropertyPath(false);
         };
 
+        self.canHaveAllowedValues = function () {
```

Comment:
This ought to be a `ko.computed` because it depends on an observable, even though `dataType` doesn't _really_ need to be an observable, since its value doesn't change.